modules/ROOT/pages/sap/sap-connector-reference-42.adoc

```diff
@@ -0,0 +1,836 @@
+* Queued (qRFC) - A queued RFC is an extension of tRFC that ensures that individual steps are processed in sequence. Use to guarantee that multiple LUWs (logical unit of work transactions) are processed in the order specified by the application. A tRFC can be serialized using queues (inbound and queues). Hence the name queued RFC (qRFC). Use qRFC to extend the transactional RFC defined processing sequence. Implement qRFC to guarantee that several transactions are processed in a predefined order.
+
+==== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration to use. | | x
+| Key a| String |  The name of the iDocument to execute. |  | x
+| Content a| Binary |  The IDocument to execute. |  `#[payload]` | 
+| Version a| String |  The version on the IDoc |  | 
+| Transaction Id a| String |  The ID that identifies an RFC that's run only once. |  | 
+| Queue Name a| String |  The name of the queue on which the RFC executes. |  | 
+| Transactional Action a| Enumeration, one of:
+
+** ALWAYS_JOIN
+** JOIN_IF_POSSIBLE
+** NOT_SUPPORTED |  The type of joining action that operations can take regarding transactions. |  JOIN_IF_POSSIBLE | 
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors. |  | 
+|===
+
+
+=== For Configurations
+
+* <<outbound-config>> 
+
+==== Throws
+
+* SAP:BEGIN_TRANSACTION_ERROR 
+* SAP:COMMIT_TRANSACTION_ERROR 
+* SAP:CONFIG_ERROR 
+* SAP:CONFIRM_TRANSACTION_ERROR 
+* SAP:CONNECTION_ERROR 
+* SAP:CONNECTIVITY 
+* SAP:EXECUTION_ERROR 
+* SAP:FIELD_NOT_FOUND_ERROR 
+* SAP:INVALID_CREDENTIALS_ERROR 
+* SAP:INVALID_HOST_ERROR 
+* SAP:MISSING_BUSINESS_OBJECT_ERROR 
+* SAP:RETRY_EXHAUSTED 
+* SAP:ROLLBACK_TRANSACTION_ERROR 
+* SAP:TID_CREATION_ERROR 
+* SAP:UNKNOWN 
+* SAP:XML_PARSING_ERROR 
+
+
+== Sources
+
+[[DocumentSource]]
+=== Document Source
+
+`<sap:document-source>`
+
+
+==== Parameters
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration to use. | | x
+| Operation Timeout a| Number |  |  | x
+| Gateway Host a| String |  |  | x
+| Gateway Service a| String |  |  | x
+| Program ID a| String |  |  | x
+| Connection Count a| Number |  |  1 | 
+| Transactional Action a| Enumeration, one of:
+
+** ALWAYS_BEGIN
+** NONE |  The type of beginning action that sources can take regarding transactions. |  NONE | 
+| Transaction Type a| Enumeration, one of:
+
+** LOCAL
+** XA |  The type of transaction to create. Availability depends on the runtime version. |  LOCAL | 
+| Primary Node Only a| Boolean |  Whether this source should only be executed on the primary node when running in Cluster |  | 
```

Comment:
"a cluster" instead of "Cluster"

Cluster should probably be lowercase. There may be other instances of this.